fhir-bulkimportexport/src/main/java/com/ibm/watson/health/fhir/bulkexport/ChunkProcessor.java

```diff
@@ -53,11 +58,24 @@ public Object processItem(Object arg0) {
         int dataSize = 0;
                         
         for (Resource res : resources) {
-            String ndJsonLine = res.toString().replace("\r", "").replace("\n", "");
+            String ndJsonLine = null;
+            StringWriter writer = new StringWriter();
+            try{
+                FHIRGenerator.generator(Format.JSON).generate(res, writer);
```

Comment:
Since we are using a non-closing outputstream to generate, the start upload could occur first, and stream out to the OutputStream to COS.

Reply:
yes, the process was added simply for formatting the output and also for the purpose of trying a chunck with all components (read, process. write) in it. and I do have plan to drop the process, and use write to push the resources to COS directly. I will have a separated PR for this.